Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp

```diff
@@ -57,7 +57,7 @@ namespace AZ::Internal
 
     AZ::IO::FixedMaxPath ReconcileEngineRootFromProjectPath(SettingsRegistryInterface& settingsRegistry, const AZ::IO::FixedMaxPath& projectPath)
     {
-        // Find the engine root via the engine manifest file and project.json
+        // Find the engine root via user/project_settings.json, engine manifest and project.json
```

Comment:
The name project_settings.json is no longer used.
```suggestion
        // Find the engine root via <project-root>/user/project.json, engine manifest and project.json
```